src/plugin.rs

```diff
@@ -1,28 +1,30 @@
+//! Contains [`ProtoPlugin`].
 use bevy::app::{App, Plugin};
 
 use crate::{
     data::{ProtoData, ProtoDataOptions, ProtoDeserializer},
     prototype::{Prototype, Prototypical},
 };
 
+/// Inserts resources for loading prototypes.
 #[derive(Default)]
 pub struct ProtoPlugin {
+    /// Optional plugin configuration.
     pub options: Option<ProtoDataOptions>,
 }
 
 impl ProtoPlugin {
-    /// Specify the directory containing the prototype files
+    /// Creates a [`ProtoPlugin`], using the given path to find prototype files.
+    /// See also: [`with_dir_recursive`][`ProtoPlugin::with_dir_recursive`].
     ///
-    /// # Arguments
+    /// # Parameters
```

Comment:
Personally I'd prefer `Arguments` since that's what rustdoc uses in their [examples](https://doc.rust-lang.org/rust-by-example/meta/doc.html#doc-comments) (and it's also what CLion automatically generates for me 😅).